cli/tools/coverage.rs

```diff
@@ -212,10 +231,38 @@ impl PrettyCoverageReporter {
   }
 }
 
-pub fn filter_script_coverages(
+fn collect_coverages(dir: &PathBuf) -> Result<Vec<Coverage>, AnyError> {
+  let mut coverages: Vec<Coverage> = Vec::new();
+
+  let entries = fs::read_dir(dir)?;
+  for entry in entries {
+    let json = fs::read_to_string(entry.unwrap().path())?;
+    let coverage: Coverage = serde_json::from_str(&json)?;
+
+    coverages.push(coverage);
+  }
+
+  coverages.sort_by_key(|k| k.script_coverage.url.clone());
+
+  // TODO(caspervonb) drain_filter would make this cleaner, its nightly at the moment.
+  for i in (1..coverages.len() - 1).rev() {
```

Comment:
Wasn't sure if this was prone to overflowing or not when I wrote this, but it is.